hack/update-generated-clientsets.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/lib/init.sh"
+
+"${OS_ROOT}/hack/build-go.sh" Godeps/_workspace/src/k8s.io/kubernetes/cmd/libs/go2idl/client-gen
+
+# Find binary
+clientgen="$(os::build::find-binary client-gen)"
+
+if [[ ! "$clientgen" ]]; then
+  {
+    echo "It looks as if you don't have a compiled client-gen binary"
+    echo
+    echo "If you are running from a clone of the git repo, please run"
+    echo "'./hack/build-go.sh Godeps/_workspace/src/k8s.io/kubernetes/cmd/libs/go2idl/client-gen'."
+  } >&2
+  exit 1
+fi
+
+# Acquire the version string used as a suffix for the client_set
+origin_version=$(cd $OS_ROOT && git describe --abbrev=0 --tags | sed -e "s/^v\([0-9]*\)\.\([0-9]*\).*/\1_\2/")
+
+packages=(
+  github.com/openshift/origin/pkg/authorization
+  github.com/openshift/origin/pkg/build
+  github.com/openshift/origin/pkg/deploy
+  github.com/openshift/origin/pkg/image
+  github.com/openshift/origin/pkg/oauth
+  github.com/openshift/origin/pkg/project
+  github.com/openshift/origin/pkg/route
+  github.com/openshift/origin/pkg/sdn
+  github.com/openshift/origin/pkg/template
+  github.com/openshift/origin/pkg/user
+  github.com/openshift/origin/pkg/security
+)
+
+
+function generate_clientset_for() {
+  local package="$1";shift
+  local name="$1";shift
+  pushd ${OS_ROOT} >/dev/null
+  local common_args="--go-header-file=hack/boilerplate.txt"
```

Comment:
@soltysh already in hack/ (I think it is empty, conversion generator use this as well)